apps/design-system-docs/README.md

```diff
@@ -0,0 +1,25 @@
+# Automattic Design System Website
+
+This website is built using [Docusaurus](https://docusaurus.io/).
+
+### Installation
+
```

Comment:
Let's also clarify that before running those commands, the dev needs to navigate to the app dir:

```
cd apps/design-system-docs
```

Reply:
Good catch e9b893a4e6c9d83445eab6050924f7b1586e3c5b